backend/src/main/java/ai/verta/modeldb/utils/ModelDBHibernateUtil.java

```diff
@@ -561,109 +562,37 @@ public static Query getWorkspaceEntityQuery(
    */
   @SuppressWarnings("unchecked")
   public static void runMigration(DatabaseConfig config)
-      throws ClassNotFoundException, ModelDBException {
+      throws ClassNotFoundException, ModelDBException, DatabaseException, SQLException {
     RdbConfig rdb = config.RdbConfiguration;
 
-    App app = App.getInstance();
-    Map<String, Map<String, Object>> migrationTypeMap =
-        (Map<String, Map<String, Object>>) app.getPropertiesMap().get(ModelDBConstants.MIGRATION);
-    if (migrationTypeMap != null && migrationTypeMap.size() > 0) {
-      new Thread(
-              () -> {
-                CommonUtils.registeredBackgroundUtilsCount();
-                int index = 0;
-                try {
-                  CompletableFuture<Boolean>[] completableFutures =
-                      new CompletableFuture[migrationTypeMap.size()];
-                  for (String migrationName : migrationTypeMap.keySet()) {
-                    Map<String, Object> migrationDetailMap = migrationTypeMap.get(migrationName);
-                    if ((boolean) migrationDetailMap.get(ModelDBConstants.ENABLE)) {
```

Comment:
this `enable` flag is not available in new config.

Reply:
I'll add it back. Ideally, we only have safe migrations and we control them by code, but that hasn't always been the case.

Reply:
Yes there is some code changes which we have to `enable` and `disable` then it would helps to manage by just config parameter.